app/fissile.go

```diff
@@ -772,7 +773,7 @@ func compareHashes(v1Hash, v2Hash keyHash) *HashDiffs {
 
 // GenerateKube will create a set of configuration files suitable for deployment
 // on Kubernetes
-func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registry, organization, fissileVersion string, defaultFiles []string, useMemoryLimits bool, opinions *model.Opinions) error {
+func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registry, organization, fissileVersion string, defaultFiles []string, useMemoryLimits bool, createHelmChart bool, chartFilename string, opinions *model.Opinions) error {
```

Comment:
Maybe, but it feels like a hack to me. Note that `--chart-file` is the name of a file (normally `Chart.yaml` that is being copied into the generated chart (directory).